jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/traceformat/TraceFormat.java

```diff
@@ -363,10 +363,10 @@ public static void main(String[] args) throws Exception {
 				if (processedMbytes % 10 == 0) {
 					int percent = (int) ((processedMbytes * 100) / (totalBytes/(1024*1024)));
 					if (verbose.booleanValue()) {
-						end = System.currentTimeMillis();
+						end = System.nanoTime();
 						float MbpsBlock = (float) 10.0 / (float) ((end - startBlock) / 1000.0);
```

Comment:
This calculation, and the next line, is done for ms.

Reply:
Fixed, pls have another look.